ote/resources/public/css/styles.css

```diff
@@ -21,6 +21,7 @@ body {
 h1, h2, h3, h4, h5, h6,
 .h1, .h2, .h3, .h4, .h5, .h6 {
     margin-bottom: 1rem;
+    font-family: 'Montserrat', montserrat, 'Source Sans', Arial, sans-serif;
```

Comment:
On a second thought, it's better to not include source sans here, because you'd have to download it from google fonts also. You could use this:
font-family: 'Montserrat', 'montserrat', sans-serif;